actions.yaml

```diff
@@ -0,0 +1,2 @@
+show-environment-context:
+  description: List variables available in StoredState
```

Comment:
I would suggest we describe this in terms that make sense for the operator. Maybe something like "List variables available for use in the `environment` config option"? And maybe we should also add a reference to this action in the description for that config option.